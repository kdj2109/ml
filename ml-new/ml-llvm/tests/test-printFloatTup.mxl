void printFloatTuples(float[] x,  int len) {
  int i;
  float f;
  for (i = 0; i < len; i = i + 1) {
    f = $x;
    printf(f);
    x = x.+;
  }
}

int main(){
	float[3] tup;
	tup = [2.3,3.4,4.5];

	printFloatTuples(@tup, tup.length);

	return 0;
}
