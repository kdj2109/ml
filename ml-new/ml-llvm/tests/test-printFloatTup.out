2.300000
3.400000
4.500000
